woocommerce/woocommerce-ios
# Improved VoiceOver support in Product Price

Fixes #2160 

## Description
Previously, as shown in [this video](https://drive.google.com/file/d/1033QAq6kB7b0Dn9obi2G4SxKevcslBLZ/view?usp=sharing), it can be challenging to understand what's happening in Product Price screen because VoiceOver is reading things out of sequence (hear for example how the price was pronounced).
In this PR I fixed the problem, managing it. Now the entire cell (price and sale price) is read as a unique component.

## Testing
1. Enable VoiceOver or use the Accessibility Inspector
2. Navigate to the price of product detail.
3. Make sure that the VoiceOver pronunciation works as expected, saying -> `Price 250.50 dollars`.
4. Try the VoiceOver with an empty price or sale price. -> The VoiceOver says `Sale Price empty`

Update release notes:
